tsconfig.base.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "rootDir": ".",
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "target": "es2015",
+    "module": "esnext",
+    "lib": ["es2020", "dom"],
+    "skipLibCheck": true,
+    "skipDefaultLibCheck": true,
+    "baseUrl": ".",
+    "paths": {
+      "react": ["packages/react/src/index.ts"],
+      "themes": ["packages/themes/src/index.ts"]
+    }
+  },
+  "exclude": ["node_modules", "tmp", "dist"]
+}
```

Comment:
Finner ingenting her om "composite" som skulle vert påkrevd for [typescript project linking](https://nx.dev/concepts/typescript-project-linking).

Men det kan vi legge til senere.